### PR TITLE
Makes sunglasses and other shielded not protect from welding

### PR DESCRIPTION
**Pull request by ivanmixo**
## About The Pull Request

Title. Also touches up some very minor errors.

## Why It's Good For The Game

Makes shielded sunglasses less meta. The abductor and experimental welding tools are still less harmful to the eyes because they should be. Fran also said this good.

## Changelog
:cl:
balance: sunglasses and other shielded glasses don't protect from welding anymore
/:cl:

### CODE REVIEW

**Comment by williamcll** [comment]
Didn't even know those existed because I just go for welding mask every time.

**Comment by PowerfulBacon** [comment]
\>Why its good for the game
\>Fran also said this good.

**Comment by Lagomorphica** [comment]
>  The abductor and experimental welding tools are still less harmful to the eyes because they should be.

Will these bypass shielded glasses or will they not? Or does it just do less eye damage?
If possible it might also be worth making sunglasses reduce the amount of eye damage taken even with welding torches, albeit I am unsure of the exact code feasibility. Perhaps making it so that advanced welding tools still blind you but don't deal any damage with glasses, but I don't code, so.

Edit: Another idea would be to deal eye damage through glasses without the flash.

**Comment by ivanmixo** [comment]
What I meant is you can still weld with shielded glasses as you could before with the abductor/experimental tool.

**Comment by BrumalLake** [comment]
you have to wear a blindfold to weld with thermal vision safely

File: code/game/objects/items/tools/weldingtool.dm [hunk 1]
```diff
@@ -255,7 +255,7 @@
 	. = ..()
 	if(. && user)
 		if (progress_flash_divisor == 0)
-			user.flash_act(min(light_range,1))
+			user.flash_act(min(light_range, 2))
```
Comment (francinum) [inline code/game/objects/items/tools/weldingtool.dm]: Why is this being clamped anyways?

Comment (ivanmixo) [reply code/game/objects/items/tools/weldingtool.dm]: I have no clue

Comment (ivanmixo) [reply code/game/objects/items/tools/weldingtool.dm]: It's likely to make sure it doesn't somehow end up being 0

Comment (francinum) [reply code/game/objects/items/tools/weldingtool.dm]: then it should be `max()`
```suggestion
			user.flash_act(max(light_range, 1))
```

**Comment by ivanmixo** [comment]
Wait what, it's already like this on master. When did that happen?

**Pull request closed without merging**